spec/lib/additional_dependents_pdf_spec.rb

```diff
@@ -0,0 +1,150 @@
+require "rails_helper"
+
+describe AdditionalDependentsPdf do
+  include PdfSpecHelper
+  let(:pdf) { described_class.new(submission) }
+  let(:submission) { create :efile_submission }
+  let(:fake_xml_document) do
+    Nokogiri::XML::Builder.new(encoding: 'UTF-8') do |xml|
+      xml.IRS1040ScheduleLEP do
```

Comment:
This is a nice way to build XML in tests that still gives you syntax highlighting.